tests/test_playbooks/tasks/upload.yml

```
@@ -14,6 +14,7 @@
     src: "{{ upload_src }}"
     repository: "{{ upload_repository }}"
     product: "{{ upload_product }}"
+    ostree_repository_name: "{{ upload_ostree_repository_name }}"
```

```suggestion
    ostree_repository_name: "{{ upload_ostree_repository_name | default(omit) }}"
```

that way it's not passed for non-ostree uploads